include/CSVAtomicReader.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef __READERS_H__
+#define __READERS_H__
+
+#include "AtomicDataReader.hpp"
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+class CSVAtomicReader : public AtomicDataReader {
+public:
+  CSVReader(std::string filepath) : AtomicDataReader(filepath) { Read(); };
```

Comment:
```suggestion
  CSVAtomicReader(std::string filepath) : AtomicDataReader(filepath) { Read(); };
```
Think this is why the declaration of content isn't causing problems - the compiler(s) might actually be ignoring this function altogether